script/library.go

```diff
@@ -7,6 +7,10 @@ import (
 // Library is a map of identifiers to underlying golang function implementations.
 type Library map[string]Func
 
+func (lib Library) Compile() starlark.StringDict {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
exported method Library.Compile should have comment or be unexported